versions.lock

```diff
@@ -1,31 +1,39 @@
 # Run ./gradlew --write-locks to regenerate this file
-ant:ant:1.6.5 (1 constraints: bb0a41d8)
-antlr:antlr:2.7.7 (2 constraints: 36167e02)
+ant:ant:1.6.5 (2 constraints: 2d1539e1)
 aopalliance:aopalliance:1.0 (1 constraints: 170a83ac)
-asm:asm:3.1 (2 constraints: 4f19c3c6)
+asm:asm:3.1 (3 constraints: 251fd7ad)
+asm:asm-commons:3.1 (1 constraints: 9c0f1f7a)
+asm:asm-tree:3.1 (1 constraints: 2307035c)
+ch.qos.logback:logback-classic:1.0.9 (3 constraints: d3258a88)
+ch.qos.logback:logback-core:1.0.9 (4 constraints: dd32435a)
+co.cask.tephra:tephra-api:0.6.0 (3 constraints: 0828ded1)
+co.cask.tephra:tephra-core:0.6.0 (2 constraints: 831cd90d)
+co.cask.tephra:tephra-hbase-compat-1.0:0.6.0 (1 constraints: 370d6920)
 com.carrotsearch:hppc:0.7.2 (1 constraints: f70cda14)
 com.clearspring.analytics:stream:2.7.0 (1 constraints: 1a0dd136)
 com.esotericsoftware:kryo-shaded:4.0.2 (2 constraints: b71345a6)
 com.esotericsoftware:minlog:1.3.0 (1 constraints: 670e7c4f)
 com.fasterxml.jackson.core:jackson-annotations:2.10.2 (5 constraints: 4155160f)
 com.fasterxml.jackson.core:jackson-core:2.10.2 (6 constraints: bb52b302)
-com.fasterxml.jackson.core:jackson-databind:2.10.2 (10 constraints: 8a92f4c4)
+com.fasterxml.jackson.core:jackson-databind:2.10.2 (11 constraints: 7c9eca9a)
 com.fasterxml.jackson.module:jackson-module-paranamer:2.10.2 (1 constraints: 03162c16)
 com.fasterxml.jackson.module:jackson-module-scala_2.11:2.10.2 (1 constraints: 7f0da251)
 com.github.ben-manes.caffeine:caffeine:2.7.0 (1 constraints: 0b050a36)
+com.github.joshelser:dropwizard-metrics-hadoop-metrics2-reporter:0.1.2 (1 constraints: e90b08f3)
 com.github.luben:zstd-jni:1.3.2-2 (1 constraints: 760d7c51)
 com.github.stephenc.findbugs:findbugs-annotations:1.3.9-1 (1 constraints: 6d05ab40)
-com.google.code.findbugs:jsr305:3.0.2 (9 constraints: d276cf3c)
-com.google.code.gson:gson:2.2.4 (1 constraints: 8c0d3f2f)
+com.google.code.findbugs:jsr305:3.0.2 (11 constraints: 2290ea31)
```

Comment:
This dependency must be excluded because it has a questionable license. The stephenc.findbugs version above is a drop-in replacement.